samples/detect.v2beta1.js

```diff
@@ -0,0 +1,823 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const projectId = process.env.GCLOUD_PROJECT;
+const sessionId = require('uuid/v1')();
+const common = require('@google-cloud/common');
+const structjson = require('./structjson.js');
+
+function createKnowledgeBase(projectId, displayName) {
+  // [START dialogflow_create_knowledge_base]
+  // Imports the Google Cloud DialogFlow library
```

Comment:
`"Google Cloud DialogFlow"` is not an approved product name (also the F should be lowercase: `Dialogflow`)

Hmm. Let's use `"DialogFlow`" in the PR

Whereas we normally has this: `// Imports the Google Cloud client library`

Let's do this: `// Imports the Dialogflow client library` <– <– <–